### PR TITLE
Changelog v1.12.0-rc2

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,16 +4,16 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
-## [1.12.0-rc1](https://github.com/go-gitea/gitea/releases/tag/v1.12.0-rc1) - 2020-05-18
+## [1.12.0-rc2](https://github.com/go-gitea/gitea/releases/tag/v1.12.0-rc2) - 2020-06-08
```
Comment (techknowlogick) [inline CHANGELOG.md:7]: Thanks for PR :)
For RC changlogs they are usually separated by RC so users can see what has changed in each individual RC, then when stable comes then they are combined.

Comment (6543) [reply CHANGELOG.md:7]: uff OK will seperate

**Comment by 6543** [comment]
@techknowlogick done

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,16 +4,70 @@ This changelog goes through all the changes that have been made in each release
...
 
+## [1.12.0-rc2](https://github.com/go-gitea/gitea/releases/tag/v1.12.0-rc2) - 2020-06-08
+
+* BUGFIXES
+  * In File Create/Update API return 404 if Branch does not exist (#11791) (#11795)
+  * Fix doer of rename repo (#11789) (#11794)
+  * Initialize SimpleMDE when making a code comment (#11749) (#11785)
+  * Fix timezone on issue deadline (#11697) (#11784)
+  * Fix to allow comment poster to edit or delete his own comments (#11671) (#11774)
+  * [API] on 500 error only show message if gitea in dev mode (#11641) (#11753)
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
  * Show full 500 error in API when Gitea in dev mode (#11641) (#11753)
```

**Review by techknowlogick — APPROVED** [review]
LGTM

One suggestion, but not sure if my suggestion is best way to phrase that ticket title.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,16 +4,70 @@ This changelog goes through all the changes that have been made in each release
...
+  * Prevent empty query parameter being set on dashboard (#11561) (#11604)
+  * Fix images in wiki edit preview (#11546) (#11602)
+  * Allow different HardBreaks settings for documents and comments (#11515) (#11599)
+  * Prevent (caught) panic on login (#11590) (#11597)
+  * Prevent transferring repos to invisible orgs (#11517) (#11549)
+  * Move serviceworker to workbox and fix SSE interference (#11538) (#11547)
+  * API PullReviewComment HTMLPullURL should return the HTMLURL (#11501) (#11533)
+  * Fix repo-list private and total count bugs (#11500) (#11532)
+  * Fix form action template substitutions on admin pages (backport #11519) (#11531)
+  * Fix the bug that the emoji don't disappear. Backport(#11489) (#11530)
```
Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
  * Fix a bug where the reaction emoji doesn't disappear. (#11489) (#11530)
```

**Review by lafriks — APPROVED** [review]

**Pull request merged**